src/backend/marsha/core/views.py

```diff
@@ -530,6 +531,7 @@ def _get_app_data(self, lti):
                 "new_video_url": new_video_url,
                 "documents": documents,
                 "videos": videos,
```

Comment:
For the bbb app independance, we probably need to split JWT tokens... one type for videos, one type for documents, one type for bbb... ping @lunika @mbenadda

Reply:
As many refactoring is needed to nicely split resources management, I think we should start this in a separate PR

Reply:
Yes of course. I was just pointing it for our brainstormings.